README.md

```diff
@@ -89,7 +89,7 @@ We also want to show some of the awesome animal sightings our traveler had. You'
 
 **YOUR NOTES**
 ```
-
+setting the data-id was the hardest (would it have been easier without the dash? not sure.) So i had to use a method called setAttribute(att, val)
 ```
```

Comment:
Attributes that begin with `data-` are called data attributes and they give us a standard way to set custom attributes on elements. [Check out this article about them](https://hacks.mozilla.org/2012/10/using-data-attributes-in-javascript-and-css/). I left a comment on your solution as well.